build/root/usr/local/bin/ci_mirror.repo

```diff
@@ -0,0 +1,51 @@
+[rhocp-4.5-for-rhel-8-x86_64-rpms]
```

Comment:
can you move this file out of `build/root/usr/local/bin`? maybe `build/root/etc` to start with (in the short term I want to reorganize `build/` into more "logical names"that `usr/...` )

Reply:
it could be that it becomes a new `subproject`, WDYT? (cc @ArangoGutierrez )